lib/text.rb

```diff
@@ -0,0 +1,20 @@
+require 'rubygems'
+require 'twilio-ruby'
+
+class Text
+    # credentials hidden
+    account_sid = 'AC86bdxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
```

Comment:
Look into use environment variables to store this data - will also mean that anyone can clone your repo and use it with their own Twilio information.